src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/home_work";
+    private static final Properties DB_PROPERTIES = new Properties();
+
+    static {
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "123456789");
```

Comment:
Storing database credentials directly in the code is not secure. Consider using environment variables or a secure vault to manage sensitive information like database usernames and passwords.